Mapping/ClassMetadata.php

```diff
@@ -106,6 +115,22 @@ public function getIri()
     }
 
     /**
+     * @return string
+     */
+    public function getIdentifier()
+    {
+        return $this->identifier;
```

Comment:
I think we must keep the code for annotation and the code for Doctrine separate:
- some people will use Doctrine metadata to determine the identifier (no annotation)
- some people will have custom data providers not using doctrine and will use the annotation to specify the id if it's not "id"
- some people will use doctrine + annotation

However, maybe that the annotation is a overkill. Having a `ClassMetadata` with an `identifier` property defaulting to `id`, using Doctrine metadata to change it automatically if necessary and requiring people not using Doctrine nor `id` as identifier to create a custom Metatadata loader looks sufficient and less complex.